sdcm/results_gemini.html

```diff
@@ -48,15 +49,31 @@ <h3>
         </tr>
         {% for res in results %}
             {% for key, value in res.items() %}
-            <tr>
-                <td class='gemini_field'>
-                    <span>{{ key }}</span>
-                </td>
-                <td>
-                    <span>{{ value }}</span>
-                </td>
-
-            </tr>
+                {% if key == "errors" %}
+                    <td class='gemini_field'>
+                        <span>{{ key }}</span>
+                    </td>
+                    <td>
+                        <ul>
+                        {% for error in value %}
+                            <li>{{ error.message }} </li>
+                            {% if loop.index > 10 %}
+                                <div>... more error messages. See <a href="{{ build_url }}">Job log</a> for details</div>
```

Comment:
can you add how many more messages there are and improve the sentance:
`... 22 more error messages. For more details see <a href="{{ build_url }}">Job log</a>.`

Reply:
Done. Redirect u mail with changes